static/code/stackblitz/v8/html/index.ts

```diff
@@ -21,6 +21,17 @@ import '@ionic/core/css/display.css';
 /* Theme variables */
 import './theme/variables.css';
```

Comment:
I think we want this file to come after the import for the dark theme so it will properly override that file if desired.

Reply:
Fixed in https://github.com/ionic-team/ionic-docs/pull/3378/commits/5b4cfed4bb75a63a29d86ca7c389264c3e9e75c7